src/client/components/Button/ButtonComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ButtonComponent.styles.css';
+
+export default function ButtonComponent({ title, backgroundColor, onClick }) {
+  return (
+    <div className="button-component">
+      <button
+        type="button"
+        onClick={onClick}
+        className="button button--primary button--medium"
+        style={{ backgroundColor }}
+      >
+        {title}
+      </button>
+    </div>
+  );
+}
+
+ButtonComponent.propTypes = {
+  title: PropTypes.string.isRequired,
+  onClick: PropTypes.func,
+  backgroundColor: PropTypes.string,
+};
+
+ButtonComponent.defaultProps = {
+  onClick: null,
+  backgroundColor: 'purple',
```

Comment:
If you want to use a default color, rather than purple, you could use the color that the wireframes indicate: #8E0EF2

Reply:
okay .. thank you